implicits/src/com/rallyhealth/weepickle/v1/implicits/MacroImplicits.scala

```diff
@@ -329,15 +329,19 @@ object MacroImplicits {
 
     def applyDefaultsWhenMissing(args: Seq[Argument]): Tree =
       q"""..${
-        for (arg <- args if arg.hasDefault) yield
-          q"if ((found & (1L << ${arg.i})) == 0) {found |= (1L << ${arg.i}); storeAggregatedValue(${arg.i}, ${arg.default})}"
+        for (arg <- args if arg.hasDefault)
+          yield q"if (isMissing(${arg.i})) {setFound(${arg.i}); storeAggregatedValue(${arg.i}, ${arg.default})}"
       }"""
 
     override def wrapObject(t: Tree) = q"new ${c.prefix}.SingletonR($t)"
 
-    override def wrapCaseN(companion: Tree, args: Seq[Argument], targetType: Type, varargs: Boolean): Tree = {
-      if (args.size > 64) {
-        c.abort(c.enclosingPosition, "weepickle does not support serializing case classes with >64 fields")
+    def wrapCaseN(companion: c.Tree, args: Seq[Argument], targetType: c.Type, varargs: Boolean) = {
+
+      val addlBitSets = math.max(0, (args.size - 1) / 64) // number of additional bitsets, excluding `CaseR.found`
+
+      def foundBitSet(i: Int) = i match {
+        case 0 => TermName("found")
+        case i => TermName(s"found$i")
       }
```

Comment:
Is this fast? (specifically for the "normal" case with << 64 fields)

Reply:
Code emitted for <= 64 fields is exactly the same as before. All the new stuff goes no-op, so perf should be identical.

I didn't even bother to bench > 64 since it's such a rare case.